### PR TITLE
Apply the Umbraco logo to BackOffice

### PR DESCRIPTION
**Pull request by nielslyngsoe**
**To create brand awareness we are adding the Umbraco logo in BackOffice 8 and 9**
The Login screen will be getting the logomark + logo name. Located clearly, but still respectful, in the top left corner.
This is where the perception of the logomark is established. In other words, the user learns the logomark stands for Umbraco.

When logged in the logomark stays in exact same position without the logo name. In this way, there is a good connection between the two logos.

**Technical details:**
Login screen still allows for overwriting logo-file through configuration(so still supporting existing feature). In BackOffice logomark is not replaceable by config, unless logo-file or code is manipulated.

image

image

Clicking the logomark in BackOffice opens a little modal which has a little more information about the version of BackOffice including a link to Umbraco.com

image

### CODE REVIEW

File: src/Umbraco.Web.UI.Client/src/less/components/application/umb-app-header.less [hunk 1]
```diff
@@ -2,12 +2,65 @@
...
+    button {
+        img {
+            height: 30px;
+        }
+    }
+}
+
+.umb-app-header__logo-modal {
+    position: absolute;
+    z-index: 7500;
```
Comment (bjarnef) [inline src/Umbraco.Web.UI.Client/src/less/components/application/umb-app-header.less:21]: @nielslyngsoe I think we should place this together with most other variables regarding z-indexes.
See also this regarding z-index of notifications and overlays.

Comment (nielslyngsoe) [reply src/Umbraco.Web.UI.Client/src/less/components/application/umb-app-header.less]: @bjarnef thanks, good catch

**Comment by nikolajlauridsen** [comment]
I think there might be missing a an svg file, when I load the backoffice and click on the icon in the top left, I get a broken image:

image

And I also a 404 for `https://localhost:44331/umbraco/assets/img/application/umbraco_logo_large_blue.svg`

**Comment by nielslyngsoe** [comment]
> I think there might be missing an svg file, when I load the backoffice and click on the icon in the top left, I get a broken image:
> 
> image
> 
> And I also a 404 for `https://localhost:44331/umbraco/assets/img/application/umbraco_logo_large_blue.svg`

Oh, yes. I need to force commit assets, sorry.

It's there now.

**Review by nikolajlauridsen — APPROVED** [review]
Looks all good to me now 👍 🎉

**Comment by bjarnef** [comment]
@nielslyngsoe any reason to have the `umbraco_logo_large_blue.svg` logo and not just a general `umbraco_logo_blue.svg`

File: src/Umbraco.Web.UI.Client/src/common/directives/components/application/umbappheader.directive.js [hunk 2]
```diff
@@ -84,6 +84,35 @@
...
+            }
+            scope.hideLogoModal = function() {
+                $timeout.cancel(scope.logoModal.timer);
+                scope.logoModal.timer = $timeout(function () {
+                    scope.logoModal.show = false;
+                }, 100);
+            }
+            scope.stopClickEvent = function($event) {
+                $event.stopPropagation();
+            }
```
Comment (bjarnef) [inline src/Umbraco.Web.UI.Client/src/common/directives/components/application/umbappheader.directive.js:114]: Ideally these named funcitons should be closed with `;` ... but I think most minifiers and modern browsers can handle this.

**Comment by patrickdemooij9** [comment]
Sorry to jump in here, but is there a reason why this logo cannot be overwritten within the config like the one on the login page?

**Comment by nielslyngsoe** [comment]
@bjarnef I named the logo with 'large' to indicate it's the logomark with logoname beneath layout logo. I guess I missed better words to describe, but as the logo-name is embedded this is not the same as the other logos. (the other logos are just the logomark).

The rest is corrected now. :-) Thanks

**Comment by nielslyngsoe** [comment]
@patrickdemooij9 

The purpose of this addition was to make Content Managers aware of the software they are using, this would sort of lose its purpose if we made it easy to change/overwrite.

We want Content Managers to know they are working in Umbraco so they can tell others about their good experience, or themself request Umbraco in the future.

From a design/solution perspective, I would like to ask if you have other suggestions on how to make it happen in a calm/respectful way? So Content Managers that don't visit the Help menu, can get an awareness of the software they are using.

Thanks

**Comment by mastrup** [comment]
I think that it could be done in a nicer way without interfering with the sections.
It has no other function than branding (the version number can be found under Help) and it takes up space for important things. Placing the logo on the right hand side, would in my opinion be a better solution.

Søren Kottal posted a nice alternative on Twitter:
chrome_eywzELm6LT

**Pull request merged**